lib/mixins/readable.js

```diff
@@ -6,7 +6,7 @@ var request = require('../request')
 
 var Readable = {
   get: {
-    value: function (id) {
+    value: function (id, query) {
```

Comment:
Yeah, maybe eventually. For now, we have to rely on tests as the way of documenting the API.